### PR TITLE
Sensor location3

### PR DESCRIPTION
**Pull request by tzachi-dar**
Code for collecting sensor locations.

### CODE REVIEW

File: app/src/main/java/com/eveningoutpost/dexdrip/StopSensor.java [hunk 1]
```diff
@@ -44,9 +44,11 @@ public void addListenerOnButton() {
         button.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
                 Sensor sensor = Sensor.currentSensor();
-                sensor.stopped_at = new Date().getTime();
-                Log.w("NEW SENSOR", "Sensor stopped at " + sensor.stopped_at);
-                sensor.save();
+                if(sensor != null) {
```
Comment (StephenBlackWasAlreadyTaken) [inline app/src/main/java/com/eveningoutpost/dexdrip/StopSensor.java:47]: Good thinking!

File: app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+        buttonCancel = (Button)findViewById(R.id.saveSensorLocationCancel);
+        sensor_location_other = (EditText) findViewById(R.id.edit_sensor_location);
+        sensor_location_other.setEnabled(false);
+        DontAskAgain = (CheckBox)findViewById(R.id.sensorLocationDontAskAgain);
+        radioGroup = (RadioGroup) findViewById(R.id.myRadioGroup);
+        addListenerOnButton();
+        
+        locations = new LinkedList<Location>();
+        
+        locations.add(new Location("private", PRIVATE_ID));
```
Comment (StephenBlackWasAlreadyTaken) [inline app/src/main/java/com/eveningoutpost/dexdrip/NewSensorLocation.java]: Can we change this text to 'I dont wish to share' or something like that

**Comment by StephenBlackWasAlreadyTaken** [comment]
I dont see the migration in here that adds the column

File: app/src/main/java/com/eveningoutpost/dexdrip/StopSensor.java (hunk 1 above)
Comment (tzachi-dar) [inline app/src/main/java/com/eveningoutpost/dexdrip/StopSensor.java:47]: Well, I have started and stopped so many sensors, that I was able to stop a sensor twice, and then it crashed...

**Comment by tzachi-dar** [comment]
Added the missing file for migration and also fixed text.

**Comment by StephenBlackWasAlreadyTaken** [comment]
Cool, Looks good to me, You still have a merge conflict but its probably just an import or log statement

**Comment by tzachi-dar** [comment]
Should I merge that to master or to beta 2?

**Comment by StephenBlackWasAlreadyTaken** [comment]
beta2 has already been pushed into master, so straight into master works. I meant to get all this up to the regular repo before this point but I have been slow on things lately

**Comment by tzachi-dar** [comment]
Ok, just created sensor_location4.

I'll check it tomorrow morning and if all is well, will commit it.

Thanks
Tzachi

**Comment by AdrianLxM** [comment]
Is this pull request still "open"?

**Comment by StephenBlackWasAlreadyTaken** [comment]
The other one was merged so this should be closed

**Pull request closed without merging**